MekHQ/src/mekhq/campaign/rating/FieldManualMercRevDragoonsRating.java

```diff
@@ -428,7 +428,7 @@ private void updateSkillLevel(Unit u, BigDecimal value) {
         final String METHOD_NAME = "updateSkillLevel(Unit, BigDecimal)";
 
         //Make sure this is a combat unit.
-        if ((null == u.getEntity()) || (null == u.getEntity().getCrew())) {
+        if ((null == u.getEntity()) || (u.getCrew().size() == 0)) {

```

Comment:
We could check for both null and empty, but I don't think there's any situation where crew can be null, so this should be fine.

Reply:
Yeah, first thing the getCrew() method does is initialize a new collection.